src/main/resources/frontend/src/views/AdminView/SubmissionsView.vue

```diff
@@ -144,7 +152,9 @@ const adminSubmit = async () => {
   <PopUp
       v-if="selectedSubmission"
       @closePopUp="selectedSubmission = null">
-    <SubmissionInfo :submission="selectedSubmission"/>
+    <SubmissionInfo
+      :submission="selectedSubmission"
+      @approvedSubmission="refreshSubmissions"/>
```

Comment:
Will this refresh the submissions every time? What if the user opens the modal, but then cancels and does not actually submit any changes? Will it still refresh all the submissions?

Reply:
the approvedSubmission emit is only triggered when an approval request has successfully made it to the server. Otherwise the window closes normally and the submissions are not refreshed